tests/cupyx_tests/scipy_tests/ndimage_tests/test_filters.py

```diff
@@ -238,6 +252,18 @@ def test_filter(self, xp, scp):
             testing.shaped_random((26, 3, 52, 70, 50), xp, xp.float32),
             [1, 1, 1, 1, 1], mode='nearest')
 
+    @testing.with_requires('scipy>=1.x.x')
+    @testing.numpy_cupy_allclose(atol=1e-5, rtol=1e-5)
+    def test_gaussian_filter_backend(self, xp):
+        x = testing.shaped_random((26, 3, 52, 70, 50), xp, xp.float32)
+        x_orig = x.copy()
+        backend = 'scipy' if xp is numpy else cupyx.scipy.ndimage
+        with scipy_ndimage.set_backend(backend):
+            out = scipy_ndimage.gaussian_filter(
+                x, [1, 1, 1, 1, 1], mode='nearest')
+        testing.assert_array_equal(x, x_orig)
+        return _correct_np_dtype(xp, xp.float32, out)
```

Comment:
Does the test fail without `_correct_np_dtype`?

Reply:
I wasn't able to test it locally, since I am using mac M1. I tried to pick up related patterns from `fft` tests.

Reply:
I verified on the `colab`, and tests were passing without `_correct_np_dtype` (the current diff)